src/components/SubCategoryDetails/SubCategoryDetails.scss

```diff
@@ -0,0 +1,45 @@
+@import '~style/variables';
```

Comment:
I have a bunch of comments below, all adding up to this as the final CSS (assuming a new `<div className={style.wrapper}>` at the top of `<SubCategoryDetails>`).
```scss
.wrapper {
  background: $color-white;
  padding-top: 32px;
  margin-bottom: 16px;
}

.container {
  @include container;
}

.heading {
  display: flex;
  align-items: center;
  margin-bottom: 16px;
}

.chartGrid {
  display: flex;
  flex-flow: row wrap;
}

.chartWrapper {
  width: 100%;

  @include for-tablet-portrait-up {
    width: 50%;
  }
}

.heroChartWrapper {
  width: 100%;
}

.iconWrapper {
  @include iconWrapper;
}
```